backend/src/services/certificate-template/certificate-template-dal.ts

```diff
@@ -0,0 +1,45 @@
+import { TDbClient } from "@app/db";
+import { TableName } from "@app/db/schemas";
+import { ormify, selectAllTableCols } from "@app/lib/knex";
+
+export type TCertificateTemplateDALFactory = ReturnType<typeof certificateTemplateDALFactory>;
+
+export const certificateTemplateDALFactory = (db: TDbClient) => {
+  const certificateTemplateOrm = ormify(db, TableName.CertificateTemplate);
+
+  const getCertTemplatesByProjectId = async (projectId: string) => {
```

Comment:
Add `try-catch` statements with `DatabaseError` for these queries (see PKI alerting DAL for reference) for consistency.